package.json

```diff
@@ -7,7 +7,6 @@
   "bin": "./bin/www",
   "scripts": {
     "compose:app": "docker-compose up -d --build app",
-    "compose:install": "npx cross-env DOCKER_INSTALL_DEPENDENCIES=1 npm run compose",
```

Comment:
I am not sure you should remove that one, it should probably still be there if we update a npm package to trigger reinstalling them in docker

Reply:
Okay, but the `npm run compose` called by the `npx`command is actually raising the error described in https://github.com/MediaComem/smapshot-api/issues/43

Reply:
Ah yes, you are right. Ok let’s remove it. We can always trigger a new fresh build if needed